src/Sylius/Bundle/AdminBundle/Resources/config/form/events.yaml

```
@@ -0,0 +1,302 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+sylius_ui:
+    events:
+        sylius.admin.admin_user.form_content:
+            blocks:
+                content:
+                    template: '@SyliusAdmin/AdminUser/_formContent.html.twig'
+                    priority: 10
+        sylius.admin.admin_user.form:
+            blocks:
+                content:
+                    template: '@SyliusAdmin/AdminUser/Form/_form.html.twig'
+                    priority: 10
+        sylius.admin.admin_user.first_column:
```

```suggestion
        sylius.admin.admin_user.form.first_column:
```
Not including form might create confusion or even name collision in the future.